.ci.yaml

```diff
@@ -3380,12 +3389,8 @@ targets:
       task_name: run_release_test
 
   - name: Mac_ios animation_with_microtasks_perf_ios__timeline_summary
-    recipe: devicelab/devicelab_drone
-    presubmit: false
-    timeout: 60
+    << : *IOS_BENCHMARK_TEMPLATE
```

Comment:
A single example of how this would be used for all iOS benchmarks.  There would be a `MAC_BENCHMARK_TEMPLATE`, etc.